### PR TITLE
Fixed eslint and added to build

### PR DESCRIPTION
**Pull request by tgreenwood**

### CODE REVIEW

File: projects/mercury/package.json [hunk 1]
```diff
@@ -67,17 +67,23 @@
...
+    "eslint-plugin-react-hooks": "^4.6.0",
     "express-session": "^1.17.1",
     "flow-bin": "^0.117.0",
+    "hermes-eslint": "^0.20.0",
     "history": "^5.0.0",
     "keycloak-connect": "^21.0.1",
     "node-fetch": "^2.6.1",
     "nodemon": "^2.0.2",
+    "react-test-renderer": "^18.2.0",
+    "react-tools": "^0.13.3",
```
Comment (ewelinajozwik) [inline projects/mercury/package.json]: Why this one? It looks like it is unused

File: projects/mercury/.eslintrc.json [hunk 2]
```diff
@@ -5,20 +5,27 @@
...
+    "ft-flow",
     "react-hooks",
     "jest"
   ],
   "extends": [
     "airbnb",
     "plugin:jest/recommended"
   ],
-  "parser": "@babel/eslint-parser",
+  "parser": "hermes-eslint",
```
Comment (ewelinajozwik) [inline projects/mercury/.eslintrc.json]: @babel/eslint-parser can be then removed from dependencies, right?

Comment (tgreenwood) [reply projects/mercury/.eslintrc.json]: Good point, removed. Double-checked the config one more time and cleaned it up even more

File: projects/mercury/package.json (hunk 1 above)
Comment (tgreenwood) [reply projects/mercury/package.json]: goof catch, removed and did a careful clean up

**Review by frankyhollywood — APPROVED** [review]

**Review by ewelinajozwik — APPROVED** [review]

**Pull request merged**